recipes/capnproto/build.sh

```diff
@@ -0,0 +1,8 @@
+#!/bin/bash
+
+mkdir build
+cd build
+cmake -DBUILD_TESTING=OFF -DCMAKE_CXX_FLAGS="-fPIC" -DCMAKE_INSTALL_PREFIX="$PREFIX" ../c++
+
+make
```

Comment:
It seems that their CMake config doesn't use `enable_testing`, so none of the listed above is available out of the box, though I can turn `BUILD_TESTING` back ON.